test/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -41,6 +41,16 @@ class CustomsDataStoreConnectorSpec extends SpecBase {
       }
     }
 
+    "return undeliverable email address from customs data store" in new Setup {
+      val emailResponse = EmailResponse(Some("noresponse@email.com"), Some("time"), Some(UndeliverableInformation("subject-example","ex-event-id-01","ex-group-id-01")))
+      val customsDataStoreUrl = "http://localhost:9893/customs-data-store/eori/GB12346/verified-email"
+      when[Future[EmailResponse]](mockHttpClient.GET(eqTo(customsDataStoreUrl), any, any)(any, any, any)).thenReturn(Future.successful(emailResponse))
```

Comment:
Better to give a line break before and this line to differentiate between val when and running(app) as it increases the readability by creating separation between different types of statements

Reply:
fixed